test/FMI2/utils.jl

```diff
@@ -4,15 +4,6 @@
 #
 using ZipFile
 
-function test_status_ok(status)
-    @test typeof(status) == fmi2Status
```

Comment:
type checks are not necessary (redundant), because we will check for type stable functions in the next release